haiku-apps/bemobile/bemobile-0.9.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Manage contacts and messages stored on a mobile phone"
+DESCRIPTION="BeMobile is a program to manage contacts and messages stored on \
+your mobile phone. It has been developed for Motorola C550 and L6 models but \
+should work with any phone that supports AT command set."
+HOMEPAGE="https://github.com/ytmytm/beos-bemobile"
+COPYRIGHT="2007 Maciej Witkowia"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="fea6464ceb5b570f7a533e59b24a68f9fb093006"
+SOURCE_URI="https://github.com/ytmytm/beos-mobile/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="2c903363e0725556d896ed5a15f68b77a0461b4512efd0106a2c68d8b30d4e5c"
+SOURCE_FILENAME="beos-bemobile-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="beos-bemobile-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	bemobile = $portVersion
+	app:BeMobile = $portVersion
+	"
+REQUIRES="
+	haiku
+        "
+
+BUILD_REQUIRES="
+	haiku_devel
+        "
```

Comment:
Indent one tab instead of spaces.

Reply:
It is already fixed.

Reply:
Nope. Still 8 spaces instead of a tab.

Reply:
But sir, the tabs are already there. Please check the code again.

Reply:
@arnavbhatt288, I can confirm that @humdingerb is right. There are spaces in line 27.
Can you please try the following command?
```bash
sed -n -e "27 s/ /#/gp" bemobile-0.9.recipe
```
If you get
```
########"
```
then this will mean you have spaces.
If you get no output then this will mean that you did not commit your local changes or that you did so but forgot to push them.

Reply:
I didn't notice that line.